src/controllers/request.controller.js

```diff
@@ -222,4 +226,64 @@ export default class RequestController {
       return errorResponse(res, badRequest, error.message);
     }
   }
+
+  /**
+   * @param {Request} req Node/express request
+   * @param {Response} res Node/express response
+   * @returns {Object} Response object with success message
+   * @description approves a trip request
+   */
+  static async approveTripRequest(req, res) {
+    const { tripRequestId } = req.params;
+    const status = ACCEPTED;
+    const current = req.tripRequestCurrentStatus;
```

Comment:
I see no need to define these constants. Which is better
` if (current === status) ` or  `if req.tripRequestCurrentStatus === ACCEPTED `, I think the second is more clear.

Reply:
Noted